### ISSUE TITLE
Editor support for typing

### ISSUE THREAD

**Issue by mikaelho** [omz/Pythonista-Issues#532]
Suggest adding editor support for `typing` in Python 3.6.

This would play nice with function argument completion suggested elsewhere, and additionally flag at coding time if providing a wrong type of an arg or assigning results to a wrong type.

**Comment by cclauss** [comment]
Is there another Editor (Atom, PyCharm, Sublime, VS Code) that does a good job of this?

**Comment by mikaelho** [comment]
I heard PyCharm [does it](https://www.jetbrains.com/help/pycharm/type-hinting-in-pycharm.html).

**Comment by Amegon** [comment]
Or try Notepad++ or Eclipse or Atom. Just need to enable it and configure when to start with displaying a list.
If you work at a small company or for private use, you can also try Visual Studio community edition. It collects all used input types and lists them in autocomplete. Would love that feature in all editors. And ability to run code and auto enter debug mode, or allow pdb.set_trace()

**Comment by mikaelho** [comment]
Probably good options for reference, but the hope here would be to get it for Pythonista.

**Comment by cclauss** [comment]
Agreed...  I was just trying to understand what were the best editors to look at for understanding the art of the possible.
